en/docs/services/create-your-first-service.md

```diff
@@ -1,177 +1,260 @@
 # Create Your First Service
 
-To learn how to create a service, let's look at a simple scenario. In this example, you'll create and design a service that consumes a public API that fetches the current active COVID-19 cases in the United States. Then you'll test it, deploy it, and observe its performance.
+This quick start guide walks you through the steps to create a service that gets the current COVID-19 status of a given country. It will:
 
-## Step 1: Create the service
+- Connect to the COVID-19 data API to get COVID-19 statistics.
+- Connect to the world bank data API to get population data.
+- Send a response back with further processed COVID-19 data.
 
-To create the service, follow this procedure:
+Once you develop and test the service, you can deploy it and observe its statistics to evaluate its performance.
+
+## Step 1: Create the service resources
+
+To create the API resource via which the service is invoked, follow these steps.
 
 1. Sign in to the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/).
     
-2. On the **Services** page, click **Create**. You are directed to the **Create Service** page.
+2. Go to the **Services** card and click **Get Started**. Now you are on the **Services** page, where you can try out sample services or create your own.
+    
+3. Click **Create** to create a service from scratch.
 
-3. Under **Create with Choreo**, enter `covid-stats` as the name of your service, and then click **Create**. You are directed to the low-code view to design the service.
+4. In the **Create with Choreo** card, enter the service name as `CovidStatus` and click **Create**.
 
-## Step 2: Design the service
+5. Select **GET** as the HTTP method, and enter `stats/[string country]` in the **Path** field.
 
-To design the service to fetch the current active COVID-19 cases in the United States, follow this procedure:
+    ![Resource configuration](../assets/img/services/configure-api-trigger.png){.cInlineImage-half}
+    
+    Then click **Save API**. 
+    
+## Step 2: Get COVID-19 data
 
-1. Select **GET** as the HTTP method and enter `activecases` as the relative path from the host.
+To get statistics related to COVID-19 via the **Covid19 API** connector, follow these steps:
 
-    ![Resource Configuration](../assets/img/services/configure-api-trigger.png){.cInlineImage-half}
-    
-2. Click **Save API**.  
+1. Click on the **API Calls** tab and then select on **Covid19 API** in the available connection list.
```

Comment:
Based on [1], we should say "Click the **API Calls** tab" right?
[1] https://developers.google.com/style/ui-elements?hl=en#term-tab

Also change "select on **Covid19 API**" to "select **Covid19 API**".